analyzer/Makefile

```diff
@@ -52,35 +64,50 @@ package_tu_collector: build_tu_collector package_dir_structure
 	# Copy tu_collector files.
 	cp -rp $(CC_TOOLS)/tu_collector/build/tu_collector/tu_collector $(CC_BUILD_LIB_DIR) && \
 	chmod u+x $(CC_BUILD_LIB_DIR)/tu_collector/tu_collector.py && \
-	cd $(CC_BUILD_DIR) && \
-	ln -sf ../lib/python3/tu_collector/tu_collector.py bin/tu_collector
+	cd $(CC_BUILD_BIN_DIR) && \
+	ln -sf ../lib/python3/tu_collector/tu_collector.py tu_collector
+
+build_report_hash:
+	$(MAKE) -C $(CC_TOOLS)/codechecker_report_hash build
+
+package_report_hash: build_report_hash package_dir_structure
+	cp -rp $(CC_TOOLS)/codechecker_report_hash/build/codechecker_report_hash/codechecker_report_hash $(CC_BUILD_LIB_DIR)
+
+build_report_converter:
+	$(MAKE) -C $(CC_TOOLS)/report-converter build
+
+package_report_converter: build_report_converter package_dir_structure
+	cp -rp $(CC_TOOLS)/report-converter/build/report_converter/codechecker_report_converter $(CC_BUILD_LIB_DIR) && \
+	chmod u+x $(CC_BUILD_LIB_DIR)/codechecker_report_converter/cli.py && \
+	cd $(CC_BUILD_BIN_DIR) && \
+	ln -sf ../lib/python3/codechecker_report_converter/cli.py report-converter
```

Comment:
These targets are duplicated in the root Makefile: https://github.com/Ericsson/codechecker/blob/e02baa734ab58c696170c7289477b2ebee1d5b06/Makefile#L42-L93

I think we can remove these targets from the root Makefile and use it from here. In the root Makefile we are calling the `package_analyzer` target from this Makefile: https://github.com/Ericsson/codechecker/blob/e02baa734ab58c696170c7289477b2ebee1d5b06/Makefile#L99

So in this file we can add these targets as dependencies of the `package_analyzer` target.

What do you think @whisperity

Reply:
Good idea. I was getting weird feelings about that massive duplication too.